cilium/tests/conftest.py

```diff
@@ -27,12 +30,55 @@
 OPERATOR_URL = "http://{}:{}/metrics".format(HOST, OPERATOR_PORT)
 
 PORTS = [AGENT_PORT, OPERATOR_PORT]
+CLUSTER_NAME = 'cluster-{}-{}'.format('cilium', get_tox_env())
 
 
 def setup_cilium():
-    config = os.path.join(HERE, 'kind', 'cilium.yaml')
+    run_command(["helm", "repo", "add", "cilium", "https://helm.cilium.io/"])
```

Comment:
Are there any new requirements/setup needed for running this locally?

Reply:
Someone running ddev locally should have helm and kind installed. I'll add this to a tests/readme file.

Reply:
Thanks!